frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeReplicaSection.tsx

```diff
@@ -1,17 +1,22 @@
 import * as React from 'react';
-import { FormGroup, FormSection, NumberInput } from '@patternfly/react-core';
+import { FormGroup, FormSection, NumberInput, Popover } from '@patternfly/react-core';
+import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
 import { UpdateObjectAtPropAndValue } from '~/pages/projects/types';
 import { CreatingServingRuntimeObject } from '~/pages/modelServing/screens/types';
 import { isHTMLInputElement, normalizeBetween } from '~/utilities/utils';
 
 type ServingRuntimeReplicaSectionProps = {
   data: CreatingServingRuntimeObject;
   setData: UpdateObjectAtPropAndValue<CreatingServingRuntimeObject>;
+  showInfoIcon?: boolean;
+  infoContent: string;
 };
 
 const ServingRuntimeReplicaSection: React.FC<ServingRuntimeReplicaSectionProps> = ({
   data,
   setData,
+  showInfoIcon,
+  infoContent,
```

Comment:
If these are so tightly coupled, you can remove `showInfoIcon` and choose to show the icon if `infoContent` is supplied.